lib/validation/validators/minLength.ts

```diff
@@ -1,3 +1,9 @@
-export function minLength(value: string, length: number): boolean {
+import { isArray, isString } from '../../support/Utils'
+
+export function minLength(value: unknown, length: number): boolean {
+  if (!(isString(value) || isArray(value))) {
+    return false
+  }
+
   return value.length >= length
 }
```

Comment:
Should this validator accept any value that has length property besides string and array?

Reply:
No let's stick with only string and array 👍